Documentation/cli/expr.md

```diff
@@ -119,6 +119,7 @@ Delve defines two special variables:
 
 * `runtime.curg` evaluates to the 'g' struct for the current goroutine, in particular `runtime.curg.goid` is the goroutine id of the current goroutine.
 * `runtime.frameoff` is the offset of the frame's base address from the bottom of the stack.
+* `runtime.bphitcount[X]` is the total hitcount for breakpoint X, which can be either an ID or the breakpoint name as a string.
```

Comment:
This is more for discussion, but I'm not crazy about this going into the `runtime` pseudo package we create for these special variables. It makes sense for the other two, as they are more related to the Go runtime (especially `runtime.curg`) however `bphitcount` is an internal (to Delve) related variable, so it seems odd to lump them together. Then again, is it worth users having to remember another pseudo package prefix?

Reply:
Do you have something in mind? Should we call it `delve.bphitcount`? I don't have a strong opinion about this.

Reply:
I like that, let's go with that.

Reply:
Done.